app/src/main/java/com/example/marvelapp/api/util/Extensions.kt

```diff
@@ -16,3 +17,22 @@ fun DataResponse.transformToCharacterList() : List<Character> {
     }
     return characterList
 }
+
+fun CharacterEntity.transformToLocalCharacter(): Character = run {
```

Comment:
why do you use a run here?

Reply:
I am using it without necessity, because I don't need to reference 'it'. I'll remove it and add a return